tools/pontoon/resources.md

```diff
@@ -0,0 +1,95 @@
+# Resources for translating in Pontoon
+
+## Machinery
+
+Pontoon features built-in access to different translation tools and repositories. These resources are available in the workspace and on a standalone web page.
+
+When translating, double click any machinery entry to auto-fill the editing space with the machinery translation.
+
+### History tab
+
+The history tab shows each of the suggestions and translations that have been submitted for the current string.
```

Comment:
Each is an emphasis on the individual entries. Would be more acceptable in a one-by-one context. 

I am updating with your recommendation. Good catch.